testsuite/patching/pom.xml

```diff
@@ -55,6 +55,20 @@
             <artifactId>guava</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>failureaccess</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-resolver-provider</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.google.guava</groupId>
+                    <artifactId>guava</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
```

Comment:
I think this is unnecessary, you have explicitly added before the `com.google.guava:guava` dependency, so the transitive one that could come from `org.apache.maven:maven-resolver-provider` never will be used, can you confirm from your experiments?

Reply:
faced maven error while building previously; removed them now, building successfully in local now. So, added the changes.